### DISCUSSION TITLE
Temperature bias in the Sulu and Celebes Seas

### DISCUSSION THREAD

**Discussion by gustavo-marques** [NCAR/omwg_dev#41] [Biases]
Meridional mean temperature biases in the Pacific show a column of cold water around`xh`=-240:
![image](https://github.com/NCAR/omwg_dev/assets/11339137/d525bf10-c304-4f27-a81f-e1320e884367)

I suspect this bias is in the [Sulu Sea](https://maps.app.goo.gl/xjK85yGXT9HEbBKNA), with the cold water entering via the straight highlighted below:

![image](https://github.com/NCAR/omwg_dev/assets/11339137/4cea996c-3648-4875-8445-94359f3475dd)

This temperature bias happens in cases with both hycom1 and zstar coordinates.

**Comment by klindsay28** [comment]
Here's a plot of `theta0` from our IC file at 9N from 100E to 140E:
![Screenshot 2024-04-24 at 9 28 11 PM](https://github.com/NCAR/omwg_dev/assets/15570875/3dd5c212-0f73-4fef-9c26-e6d5d81055cb)
It has water around 9.9 degC going down to 5000m. That's pretty warm.
This warm water does appear in on older [online WOA](https://sam.ucsd.edu/whp_atlas/indian/map_depths.htm), such as [theta0 at 2500m (final)](https://sam.ucsd.edu/whp_atlas/indian/maps/theta/ind2500_theta_final.jpg).
I'm not sure what to make of it.

**Reply by klindsay28** [reply]
Lynne Talley confirms that observations support this deep warm water in WOA.

**Comment by gustavo-marques** [comment]
The topo changes in https://github.com/NCAR/omwg_dev/issues/42 and https://github.com/NCAR/omwg_dev/issues/43 fixed the bias in the Sulu Sea. However, I now see a similar bias in the [Celebes Sea](https://maps.app.goo.gl/RDzrQGLfb7BeQkEA9). Here is my proposed topo changes for an upcoming simulation (055):

### Sangihe Islands (between Philippines and Indonesia)

Before:
![image](https://github.com/NCAR/omwg_dev/assets/11339137/92dad81d-0985-4cb1-9424-87d977fc6adf)


After topo changes:
![image](https://github.com/NCAR/omwg_dev/assets/11339137/18ee2483-8d51-4f1b-acc9-e0f5e3c398c6)

**Comment by gustavo-marques** [comment]
Here are meridional mean temperature biases in the Pacific for cases [049](https://github.com/NCAR/omwg_dev/issues/42) and [055](https://github.com/NCAR/omwg_dev/issues/46) over the first 20 years of simulation:

![image](https://github.com/NCAR/omwg_dev/assets/11339137/4bdeb64b-bf3a-4c80-bb20-2c2c5da742c5)

![image](https://github.com/NCAR/omwg_dev/assets/11339137/485b9f98-5aa8-4b1f-8b42-a8402583ef1d)
